### ISSUE TITLE
Serve Prometheus metrics in non-"db serve" process

### ISSUE THREAD

**Issue by philrz** [brimdata/super#5551]
## tl;dr

Right now Prometheus metrics are only made available on `http://localhost:9867/metrics` if running `super db serve`. It would be helpful if we had an option to open this up even when running `super` in regular, non-`db serve` mode, as this would allow for things like getting per-second memory/heap usage stats when studying vector runtime performance.

## Details

At the time this issue is being opened, `super` is at commit 4084e01.

When I asked the team about this, @mattnibs offered tips the code to add this as a one-off hack, and I've done this successfully on the [prometheus-hack](https://github.com/brimdata/super/tree/prometheus-hack) branch. I'll keep that branch around in case I or a user should want to apply the commit on a one-off basis. However, if someone could add this as an official flag so it could easily be enabled whenever needed that would be great.
